examples/todo/package.json

```diff
@@ -19,7 +19,7 @@
     "react": "^0.14.0-rc",
     "react-dom": "^0.14.0-rc",
     "react-relay": "file:../../",
-    "react-router": "^1.0.0-rc1",
```

Comment:
I think this *should* install rc3 as is (at least with a fresh install or `npm update`)